### PR TITLE
kronometer: name -> pname, version

### PR DESCRIPTION
**Pull request by peterhoeg**
###### Motivation for this change

Minor cleanup - should go into staging with the other kde changes.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: pkgs/tools/misc/kronometer/default.nix [hunk 1]
```diff
@@ -22,6 +19,6 @@ mkDerivation rec {
     license = licenses.gpl2;
     maintainers = with maintainers; [ peterhoeg ];
   };
-  nativeBuildInputs = [ extra-cmake-modules kdoctools wrapGAppsHook ];
+  nativeBuildInputs = [ extra-cmake-modules kdoctools ];
   propagatedBuildInputs = [ kconfig kcrash kinit ];
```
Comment (worldofpeace) [inline pkgs/tools/misc/kronometer/default.nix]: needs propagation?

**Comment by worldofpeace** [comment]
>  should go into staging with the other kde changes.

I can see how that works organizationally, but it doesn't appear to change anything in staging.
Does it need to be built in staging to check for regressions? And even then, master gets merged into staging so we'll see it newly failing.

**Comment by peterhoeg** [comment]
> but it doesn't appear to change anything in staging

No, but it will need to be rebuilt when staging hits master due to a lot of kde changes currently in staging. So we might as well just merge this as part of that and only build it once.

**Comment by worldofpeace** [comment]
Sounds reasonable then.

**Pull request merged**